### PR TITLE
Fixed pages showing up in wrong language

### PR DESCRIPTION
**Pull request by JackieNim**
Better PR of https://github.com/iluwatar/java-design-patterns/pull/1750

### CODE REVIEW

**Comment by ohbus** [comment]
Please **continue** with #1750

It is very difficult and time-consuming to review the same files again.

**Comment by JackieNim** [comment]
> 
> 
> > > @JackieNim reopen #1750 and push commits that you want to rectify.
> > > Thank you.
> > 
> > 
> > @ohbus I kinda deleted that fork so im not sure if I can do that :(
> 
> It takes time to review 180+ files.
> 
> Please try to reopen that PR and connect the fork.

Yeah it wont let me reopen it since the repo was deleted

File: double-buffer/README.md [hunk 1]
```diff
@@ -25,4 +26,4 @@ This pattern is one of those ones where you’ll know when you need it. If you h
 
 ## Credits  
   
-* [Game Programming Patterns - Double Buffer](http://gameprogrammingpatterns.com/double-buffer.html)
+* [Game Programming Patterns - Double Buffer]([http://gameprogrammingpatterns.com/double-buffer.html](http://gameprogrammingpatterns.com/double-buffer.html))
```
Comment (iluwatar) [inline double-buffer/README.md:29]: This change is probably a mistake and should be reverted

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: double-buffer/README.md (hunk 1 above)
Comment (JackieNim) [reply double-buffer/README.md]: Should be fixed now

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Many thanks @JackieNim for the fix

**Pull request merged**